postgres/upstream.sls

```diff
@@ -41,13 +39,12 @@ postgresql-repo:
     - keyid: {{ postgres.pkg_repo_keyid }}
     {%- endif %}
 
-  {%- endif -%}
-
-{%- elif grains.os not in ('Windows', 'MacOS',) %}
```

Comment:
Does this changes are really required? It seems like 

```yaml
Arch:
  use_upstream_repo: false
```

should be enough because it will disable requisite https://github.com/saltstack-formulas/postgres-formula/blob/db1794b6bbb6ce183e5231cb4b7e7193dcb80143/postgres/client/init.sls#L19-L23 mentioned in PR

Reply:
Correct it's not required.  I think I introduced the line `{%- if 'pkg_repo' in postgres -%}` as a fast fix for missing requisite some time ago when in a hurry so I doubt its best practice/solution. But I'll adopt your comment to minimize impact.